profile.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz-App</title>
+    <link rel="stylesheet" href="./CSS/styles.css" />
+  </head>
+  <body>
+    <header class="header">
+      <h1>Quiz-App</h1>
+    </header>
+
+    <main>
+      <section class="profile">
```

Comment:
Try to find a common, generalized class name for all these `<section>`'s since they most likely will be styled very similar. An donly if you need more specific styles for one of them add a "identifying" class name, as well (ie. 'about-me').